fastlane/Fastfile

```diff
@@ -123,7 +122,6 @@ platform :ios do
   lane :deploy do |options|
     version_number = current_version_number
     push_pods
-    carthage_archive
```

Comment:
we should keep the `carthage_archive`, since we're not removing _compatibility_ with carthage, we're just not using it for development anymore

Reply:
✅